PowerShell/ScubaGear/Modules/Providers/ExportAADProvider.psm1

```diff
@@ -196,25 +196,56 @@ function Get-PrivilegedUser {
             $Objecttype = $User.AdditionalProperties."@odata.type" -replace "#microsoft.graph."
 
             if ($Objecttype -eq "user") {
+                # If the user's data has not been fetched from graph, go get it
                 if (-Not $PrivilegedUsers.ContainsKey($User.Id)) {
                     $AADUser = Get-MgBetaUser -ErrorAction Stop -UserId $User.Id
```

Comment:
I spoke with @buidav about this and because it is an edge case we added a new issue to address it in the near future. With that enhancement there is a secondary opportunity to reduce redundant lines of code where we make calls to Get-MgBetaUser.